docs/APIExamples.md

```diff
@@ -206,7 +207,7 @@ layer = spec.neuralNetwork.layers[0]
 weight_params = layer.convolution.weights
```

Comment:
can we change it to a loop like:
Weights_conv = []
for layer in spec.neuralNetwork.layers:
   if layer.WhichOneof('layer') == 'convolution':
       Weights_conv.append(layer.convolution.weights)